set.mm

```diff
@@ -472151,6 +472186,240 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Sumset / Minkowski sum
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  The sumset (also called the Minkowski sum) of two subsets ` A ` and ` B ` ,
+  is defined to be the set of all sums of an element from ` A ` with an element
+  from ` B ` .
+
+  For an operation ` .+ ` This can be expressed as ` ( .+ " ( A X. B ) ) ` .
+  This notation is concise, so no new notation needs to be introduced.
+
+  General theorems exist about this construct (see e.g. ~ ovelimab ), however
+  they are not directly applicable to groups, where the group operation is not
+  guaranteed to be a function.  The theorems in this section help in that case.
+
+  The sumset operation can be used for both group (additive) operations and
+  ring (multiplicative) operations.
```

Comment:
What does that mean ?

Reply:
I meant it's call "**sum**set" but actually we can use it for any group operations, not only "sums".